types.d.ts

```diff
@@ -2006,7 +2006,7 @@ declare class Compiler {
 	constructor(context: string, options?: WebpackOptionsNormalized);
 	hooks: Readonly<{
 		initialize: SyncHook<[]>;
-		shouldEmit: SyncBailHook<[Compilation], boolean>;
+		shouldEmit: SyncBailHook<[Compilation], undefined | boolean>;
```

Comment:
You can return nothing, it means like `true`, because we have `this.hooks.shouldEmit.call(compilation) === false`